docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -0,0 +1,175 @@
+Again, you need to check for the ``Tool Name`` field being set because it's needed for multi tool repositories to ensure that the tool paths on Dockstore are unique.
+
+Let's say we want to convert these two tools that come from this `repository <https://github.com/dockstore/dockstore-tool-bamstats>`__.
+
+.. image:: /assets/images/docs/github-apps-multiple-tools.png
+
+.. image:: /assets/images/docs/github-apps-multiple-tools-with-name.png
+
+
+Your ``/.dockstore.yml`` would look like the following:
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.cwl.json
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore2.cwl
+        testParameterFiles:
+            - /test.cwl.json
+        name: optional-name
+
+Archiving or Deleting Your Existing Tool
+----------------------------------------
+
+Once you've adding a ``.dockstore.yml`` to the desired branch of your repository, you should see a new tool appear on your ``/my-tools`` page. The tool path will start with ``github.com`` and 
+you should see that the ``Tool Information`` section looks a bit different from your existing tool.
+
+.. figure:: /assets/images/docs/github-app-tool.png
+   :alt: New GitHub App tool
+
+The mode is ``Automatically synced via GitHub App`` instead of one of our three build modes, and information about paths and your Docker Image is no longer included.
+You are also not able to refresh or restub the new GitHub App tool. Since you can't refresh the entire tool anymore, **new** versions from GitHub (releases/branches) that you want to add to Dockstore must have a ``.dockstore.yml`` file.
+
+If you look on the left hand side bar, you should see that your old tool still exists. Your new GitHub App tool will start with ``github.com``, and your old tool will start with
+the Docker registry, which is ``quay.io`` in this case.
+
+.. figure:: /assets/images/docs/old-and-new-tool.png
+   :alt: Old and new tools
+   :width: 50 %
+
+At this point, you must decide whether you want to keep your old tool around. You may want to keep it if you want to preserve the link to the tool. For example, you may choose to keep the tool if this tool was linked in a research paper
+and other people might visit the tool's page.
+
+If you need to keep your old tool, then we recommend that you add a short description to your GitHub repository's README indicating where the new GitHub App tool can be found on Dockstore. 
+You may also indicate whether your old tool will be kept up to date on Dockstore through refresh. Once a short description has been added, navigate to your old tool's page and click ``Refresh`` to 
+update your tool's description with the new information.
+
+If you do not need to keep your old tool, simply delete it by navigating to your old tool and pressing the ``Delete`` button in the upper right corner.
```

Comment:
Don't you have to unpublish first (if published)? I vaguely recall some issues with this.

Reply:
Assuming publish/unpublish the same as the current release, then a tool must be unpublished for the "delete" button to show.